src/main/kotlin/domain/Game.kt

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import kotlin.random.Random
+
+class Game {
+    var carList = mutableListOf<RacingCar>()
+
+    fun makingCar(number: Int, moving: Int) {
+        repeat(number) {
+            carList.add(RacingCar(remainingMoves = moving))
+        }
+    }
+
+    private fun generateRandomNumber(): Int {
+        return Random.nextInt(0, 10)
```

Comment:
```suggestion
        return Random.nextInt(0..9)
```
범위를 주는 건 어떨까요? 그러면 0..9는 상수로도 표현이 가능하겠죠?